src/goCover.ts

```diff
@@ -305,3 +309,19 @@ export function toggleCoverageCurrentPackage() {
 		});
 	});
 }
+
+export function isPartOfComment(e: vscode.TextDocumentChangeEvent): boolean {
+	let result = false
+	e.contentChanges.every(function(change, index){
+		let line = e.document.lineAt(change.range.start)
+		let text = e.document.getText(line.rangeIncludingLineBreak)
+		// Exit the loop as soon as one of the changes is not part of a comment.
+		if (!text.startsWith('//')){
```

Comment:
What about a case like the below?

`var a = 1 // this is a comment`

Here when the comment is edited, `isPartOfComment` returns `false` instead of `true`

Reply:
Fixed. Will update the PR once I have fixed the block comment issue.